src/ParallelMessage.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/*
+ * This file is part of the prooph/humus-amqp-producer.
+ * (c) 2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare (strict_types=1);
+
+namespace Prooph\ServiceBus\Message\HumusAmqp;
+
+use Prooph\Common\Messaging\Message;
+
+/**
+ * Interface to represent delayed messages (aka messages that are processed in the future instead of now)
```

Comment:
@prolic copy & paste issue. If description reflects the idea of the interface I'm fine with the name.